### PR TITLE
misc: tidy ups pre 0.16

### PR DESCRIPTION
**Pull request by davidhewitt**
cc @mejrs @birkenfeld

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -78,6 +78,8 @@
 //! - `multiple-pymethods`: Enables the use of multiple [`#[pymethods]`](macro@crate::pymethods)
 //! blocks per [`#[pyclass]`](macro@crate::pyclass). This adds a dependency on the [inventory]
 //! crate, which is not supported on all platforms.
+//! - `pyproto`: Adds the deprecated `#[pyproto]` attribute macro. Enabled by default, and likely
```
Comment (mejrs) [inline src/lib.rs:81]: This should be under the Default feature header, together with the macros feature (whose description isnt quite right anymore 🙃)

Comment (davidhewitt) [reply src/lib.rs]: Thanks, moved 👍

**Review by mejrs — APPROVED** [review]
Thanks ❤️

**Review by birkenfeld — APPROVED** [review]
Thanks!

**Comment by davidhewitt** [comment]
Just pushed a little bit of extra coverage, then will merge & rebase release.

**Pull request merged**